src/metric_model_reporter.h

```diff
@@ -46,25 +46,30 @@ struct ModelIdentifier;
 struct MetricReporterConfig {
 #ifdef TRITON_ENABLE_METRICS
   // Parses Metrics::ConfigMap and sets fields if specified
-  void ParseConfig(bool response_cache_enabled);
+  void ParseConfig(bool response_cache_enabled, bool is_decoupled);
   // Parses pairs of quantiles "quantile1:error1, quantile2:error2, ..."
   // and overwrites quantiles_ field if successful.
   prometheus::Summary::Quantiles ParseQuantiles(std::string options);
 
   // Create and use Counters for per-model latency related metrics
   bool latency_counters_enabled_ = true;
+  // Create and use Histograms for per-model latency related metrics
+  bool latency_histograms_enabled_ = true;
```

Comment:
I think we should default to `false` until we've done some light benchmarking to verify any performance impact - especially since it looks like we're planning to cherry-pick this and planning to add multiple histogram metrics here.

After some further testing and perf validation, I think we can change the default to be enabled later if we want.

Reply:
Fixed

Reply:
Updated